# 
float32[] angle_void_min
float32[] angle_void_max
